setup.cfg

```diff
@@ -21,7 +21,7 @@ all_files = 1
 [upload_sphinx]
 upload-dir = docs/_build/html
 
-[pytest]
+[tool:pytest]
```

Comment:
maybe don't do that now.

I also did that in borg and later remove tool: again for better compatibility.